tests/foreman/ui/test_product.py

```diff
@@ -189,4 +188,22 @@ def test_positive_bulk_action_advanced_sync():
 
     :expectedresults: Advanced sync for repositories can be run as a bulk action from the product.
     """
-    pass
+    repo_name = gen_string('alpha')
+    product = entities.Product(organization=module_org).create()
+    with session:
+        session.repository.create(
+            product.name,
+            {
+                'name': repo_name,
+                'repo_type': REPO_TYPE['yum'],
+                'repo_content.upstream_url': settings.repos.yum_1.url,
+            },
+        )
+        # Repository sync
+        session.repository.synchronize(product.name, repo_name)
+        # Optimized Sync
+        result = session.product.advanced_sync([product.name], sync_type="optimized")
+        assert result["task"]["result"] == "success"
+        # Complete Sync
+        result = session.product.advanced_sync([product.name], sync_type="complete")
+        assert result["task"]["result"] == "success"
```

Comment:
Please switch these to single-quote strings.

Reply:
Okay